src/firefly/java/edu/caltech/ipac/firefly/server/query/EmbeddedDbProcessor.java

```diff
@@ -157,7 +157,7 @@ public DataGroupPart getData(ServerRequest request) throws DataAccessException {
             boolean dbFileCreated = false;
             File dbFile = getDbFile(treq);
             jobExecIf(v -> v.progress(10, "fetching data..."));
-            if (!dbFile.exists()) {
+            if (!dbFile.exists() || !EmbeddedDbUtil.hasTable(treq, dbFile, MAIN_DB_TBL)) {
```

Comment:
was this caused MOST search table not showing in the result panel?

Reply:
No.  This is the cause of `DATA not found` exception we see after redeploying an app multiple times.  You normally see this during development.  Not related to MOS.